MSAL/test/unit/native_auth/network/errors/reset_password/MSALNativeAuthResetPasswordSubmitOauth2ErrorCodeTests.swift

```diff
@@ -44,24 +44,8 @@ final class MSALNativeAuthResetPasswordSubmitOauth2ErrorCodeTests: XCTestCase {
     func test_expiredToken() {
         XCTAssertEqual(sut.expiredToken.rawValue, "expired_token")
     }
-    
-    func test_passwordTooWeak() {
-        XCTAssertEqual(sut.passwordTooWeak.rawValue, "password_too_weak")
-    }
-    
-    func test_passwordTooShort() {
-        XCTAssertEqual(sut.passwordTooShort.rawValue, "password_too_short")
-    }
-    
-    func test_passwordTooLong() {
-        XCTAssertEqual(sut.passwordTooLong.rawValue, "password_too_long")
-    }
-    
-    func test_passwordRecentlyUsed() {
-        XCTAssertEqual(sut.passwordRecentlyUsed.rawValue, "password_recently_used")
-    }
-    
-    func test_passwordBanned() {
-        XCTAssertEqual(sut.passwordBanned.rawValue, "password_banned")
+
+    func test_invalidGrant() {
```

Comment:
nit: should we put this one at the top to preserve order of tests?

Reply:
`invalidGrant` is the last case of the enum `MSALNativeAuthResetPasswordSubmitOauth2ErrorCode`.

Reply:
Ok, we can keep it there, they are a bit all over the place indeed, it would have been good to have them sorted by something but all good.